azure-pipelines.yml

```diff
@@ -7,6 +7,12 @@ trigger:
   tags:
     include:
     - 0.*
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - master
+    - 0.*_maintenance
```

Comment:
We should be able to use `- '*'` here to cover all branches. Note that the quotes are necessary because `*` has special meaning to YAML.

Reply:
Excellent idea https://github.com/GafferHQ/gaffer/pull/3670/commits/478b6ce79c4401deb95346741b8da9c8766dfcaa